src/main/java/core/basesyntax/figures/Square.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.figures;
+
+public class Square extends Rectangle {

```

Comment:
I can understand your logic but I guess it can violate the Liskov Substitution principle (soon we will study SOLID principles). I recommend you make Square independent from Rectangle.